assets/js/composer.js

```diff
@@ -299,39 +350,31 @@ PageComposer.prototype = {
   handleBlockCreateFormLoaded(event) {
     const self = this;
     const $containerChildren = this.$dynamicArea.find('.page-composer__container__children');
-    const $container = this.$dynamicArea.find('.page-composer__container__main-edition-area');
```

Comment:
https://github.com/sonata-project/SonataPageBundle/commit/e66956ec95856149a716e4bb361f91b8df6f5fc3

Lefovers from this commit...